engine/src/main/java/org/terasology/input/binds/general/LoadButton.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.input.binds.general;
+
+import org.terasology.input.BindButtonEvent;
+import org.terasology.input.DefaultBinding;
+import org.terasology.input.InputType;
+import org.terasology.input.Keyboard;
+import org.terasology.input.RegisterBindButton;
+
+@RegisterBindButton(id = "load", description = "${engine:menu#binding-load}", category = "general")
```

Comment:
I thought it would be more convenient for the user. To safeguard against this, perhaps it should lead to a screen asking for confirmation from the user if they truly want to load their quick save state?